src/library/scala/math/Ordering.scala

```diff
@@ -309,6 +312,8 @@ object Ordering extends LowPriorityOrderingImplicits {
       override def gteq(x: Double, y: Double): Boolean = outer.gteq(y, x)
       override def lt(x: Double, y: Double): Boolean = outer.lt(y, x)
       override def gt(x: Double, y: Double): Boolean = outer.gt(y, x)
+      override def min(x: Double, y: Double): Double = outer.max(x, y)
+      override def max(x: Double, y: Double): Double = outer.min(y, x)
```

Comment:
Okay, but then why _are_ the args swapped here?  Not that it really matters since practically nobody cares which NaN it is, but both min and max return the first argument's NaN even if the second argument is also NaN.